kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
@mwrock I believe this is a recurring bug (when working from a windows box?) and you need to add this line back.